This tries to download files from cmake repository, dependending on cmake
version. We don't want this to happen in robotpkg.

--- cmake-next/CMakeLists.txt~	2024-06-03 06:35:57.000000000 +0200
+++ cmake-next/CMakeLists.txt	2024-06-24 15:59:55.451346939 +0200
@@ -118,7 +118,7 @@
                              proposed/CMakeParseArguments.cmake)
 set(YCM_CMAKE_PROPOSED_EXTRA_FILES  Copyright.txt # Must be redistributed together with CMake files
                                     README)       # Overwrite README generated by _ycm_cmake_next_download
-_ycm_cmake_next_install(proposed FILES ${YCM_CMAKE_PROPOSED_FILES}
-                                 DESTINATION "${YCM_INSTALL_MODULE_DIR}/cmake-proposed/Modules")
+#_ycm_cmake_next_install(proposed FILES ${YCM_CMAKE_PROPOSED_FILES}
+#                                 DESTINATION "${YCM_INSTALL_MODULE_DIR}/cmake-proposed/Modules")
-_ycm_cmake_next_install(proposed FILES ${YCM_CMAKE_PROPOSED_EXTRA_FILES}
-                                 DESTINATION "${YCM_INSTALL_MODULE_DIR}/cmake-proposed")
+#_ycm_cmake_next_install(proposed FILES ${YCM_CMAKE_PROPOSED_EXTRA_FILES}
+#                                 DESTINATION "${YCM_INSTALL_MODULE_DIR}/cmake-proposed")
